e2e-tests/tfm/cypress/e2e/journeys/case-amendments/show-amendment-button/deal-cancellation-enabled/amendment-button.spec.js

```diff
@@ -0,0 +1,37 @@
+import relative from '../../../../relativeURL';
+import facilityPage from '../../../../pages/facilityPage';
+import amendmentsPage from '../../../../pages/amendments/amendmentsPage';
+import MOCK_DEAL_AIN from '../../../../../fixtures/deal-AIN';
+import { T1_USER_1, PIM_USER_1, BANK1_MAKER1 } from '../../../../../../../e2e-fixtures';
+
+context('Amendments page - deal cancellation feature flag enabled', () => {
```

Comment:
Not a blocker, can be done in another PR - we have this in other places

```suggestion
context('Amendments page - deal cancellation - feature flag enabled', () => {
```

Reply:
I'll pick this up in another PR